packages/models-library/tests/test_generics.py

```diff
@@ -39,20 +46,87 @@ def test_dict_base_model():
     assert some_instance["a new key"] == 23
 
 
-def test_data_enveloped(faker: Faker):
-    some_enveloped_string = Envelope[str]()
-    assert some_enveloped_string
-    assert not some_enveloped_string.data
-    assert not some_enveloped_string.error
-
-    random_float = faker.pyfloat()
-    some_enveloped_float = Envelope[float](data=random_float)
-    assert some_enveloped_float
-    assert some_enveloped_float.data == random_float
-    assert not some_enveloped_float.error
-
+def test_enveloped_error_str(faker: Faker):
```

Comment:
@sanderegg: the idea is in that line ... We only use `null` to nullify. If you pass an empty string that is indeed what you want to pass as an error or data. 

We had a bug that was produced because we were interpreting empty dictionaries as nullification of that field, but in reality it was data. 

We tend to mix objects that evaluate to `False` with nullification of a field!